packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -305,8 +307,35 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
         _getListTile(
           appLocalizations.account_delete,
           () async {
+            await showDialog<SmoothAlertDialog>(
```

Comment:
Should be `final String? reason = await showDialog<String>(`.

Reply:
> @monsieurtanuki So reason string will be used in email ?

@Akashsri3bi It is already, isn't it? If we think about something more refined for this "delete account" feature, it will be in another issue and another PR.